JitsiMeetJS.js

```diff
@@ -42,6 +43,9 @@ var LibJitsiMeet = {
 
     version: '{#COMMIT_HASH#}',
 
+    // expose JitsiTrackError this way to give library consumers to do checks
+    // like if (error instanceof JitsiMeetJS.JitsiTrackError) { }
+    JitsiTrackError: JitsiTrackError,
```

Comment:
There are a couple of things that are confusing here: 1) We have JitsiTrackError and JitsiTrackErrors. I don't understand why we need 2 different files dealing with JitsiTrackErrors or at least we need to name them differently, so that it's clear what's the exact purpose behind if it's different. 2) They're both exposed in different ways. We're already exporting errors.track.JitsiTrackErrors (see below code).